docs/surveys/salary-survey/2020.rst

```diff
@@ -0,0 +1,1876 @@
+Onsite - not required Partial               16  2.6
+Remote - not required Remote - required     13  2.11
+Other                                       9   1.46
+===================== ===================== === =====
+
+Respondents who indicated that they had experienced a work location
+change due to COVID-19 were asked if they thought that the changes would
+be permanent or temporary, and also how they felt about the change.
+
+Opinions on the permanency of the changes were quite evenly spread -
+however those who predicted “no” (25.4%) or “probably no” (22.2%) - a
+combined 47.6% - outweighed those that predicted “yes” (13.2%) or
+“probably yes” (22.9%) - a combined 36.1%.
+ 
+ .. figure:: images/2020/permenancy-location-change.svg
+   :alt: Figure: Permanency of Work Location Changes
+
+   Figure: Permanency of Work Location Changes
+ 
+While other aspects of living through a pandemic might be challenging, a
+large proportion of respondents reported finding a silver lining in work
+location changes. More than 60% of respondents reported feeling
+“positive” (34.11%) or “very positive” (26.51%) about the change, 27.29%
+felt “neutral”, and only 12.09% reported feeling “negative” (11.47%) or
+“very negative” (0.62%, or 4 individuals).
+
+.. figure:: images/2020/feelings-location-change.svg
+   :alt: Figure: Feelings About Work Location Change
+
+   Figure: Feelings About Work Location Change
+
+--------------
+
+.. _h.xvhht34qf1cm:
+
+.. _h.7btw5lpuhsu:
+
+Work Location Unchanged
+'''''''''''''''''''''''
+
+Of those respondents (20%) who indicated that their work location had
+not changed due to COVID-19, 45% were required to be remote, 38.7% were
+remote by choice, and 6.3% were partially onsite and partially remote.
+Only 10% (16 individuals) worked onsite, either by choice (5%) or
+necessity (5%).
+
+.. figure:: images/2020/work-location-unchanged.svg
+   :alt: Figure: Work Location (unchanged since COVID-19)
+
+   Figure: Work Location (unchanged since COVID-19)
+
+--------------
+
+In response to their feelings about their work location, of the 83.8%
+that worked remotely, 67.9% reported feeling “very positive” and 24.6%
+reported “positive”. 10 individuals (7.5%) were “neutral” about their
+work location, and no remote workers in this group felt at all negative
+about the situation.
+
+Similarly, no negativity was reported from the 16 respondents in this
+group who worked on-site. Half of the on-site workers felt “very
+positive” and the other half were split between “positive” and
+ “neutral”. In fact, only 1 respondent - from the “partially remote,
+partially onsite” segment - reported feeling “negative” about their work
+location, and no one reported feeling “very negative”.
+
+.. figure:: images/2020/feelings-location-unchanged.svg
+   :alt: Figure: Feelings about Work Location
+(where work location is unchanged since COVID-19)
+
+   Figure: Feelings about Work Location
+(where work location is unchanged since COVID-19)
```

Comment:
```suggestion
.. figure:: images/2020/feelings-location-unchanged.svg
   :alt: Figure: Feelings about Work Location (where work location is unchanged since COVID-19)

   Figure: Feelings about Work Location (where work location is unchanged since COVID-19)
```